### PR TITLE
{system} [intel/2016a] gnutls (REVIEW)

### PR DESCRIPTION
**Pull request by wpoely86**
@boegel

### CODE REVIEW

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 9 out of 10 (7 easyconfigs in this PR)
node2622.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/413d6d1b45b847556fce for a full test report.

File: easybuild/easyconfigs/l/libtasn1/libtasn1-4.7-intel-2016a.eb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ for some time but is now a proper GNU package."""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+
+source_urls = [GNU_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+sanity_check_paths = {
+    'files': ['bin/asn1%s' % x for x in ['Coding', 'Decoding', 'Parser']] +
+             ['lib/libtasn1.%s' % x for x in ['a', 'so', 'so.6']],
```
Comment (boegel) [inline easybuild/easyconfigs/l/libtasn1/libtasn1-4.7-intel-2016a.eb]: `s/'so'/SHLIB_EXT/g`

File: easybuild/easyconfigs/p/p11-kit/p11-kit-0.23.2-intel-2016a.eb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+sources = [SOURCE_TAR_GZ]
+
+dependencies = [
+    ('libtasn1', '4.7'),
+    ('libffi', '3.2.1'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/p11-kit', 'bin/trust'] +
+             ['lib/libp11-kit.%s' % x for x in ['so', 'so.0']],
```
Comment (boegel) [inline easybuild/easyconfigs/p/p11-kit/p11-kit-0.23.2-intel-2016a.eb]: here also `SHLIB_EXT` please

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 9 out of 10 (7 easyconfigs in this PR)
node2638.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/a4252149e74adcba33d4 for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 8 out of 8 (7 easyconfigs in this PR)
gligar01.gligar.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/e54ffee8612c858ce287 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 6 out of 7 (7 easyconfigs in this PR)
node2668.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/7df38bbf2b5ff1b10854 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 6 out of 7 (7 easyconfigs in this PR)
node2668.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/c2a4cf839415798452e6 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in this PR)
node2668.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/ccc9d0ccb8588b0c3f49 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @wpoely86!

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 6 out of 7 (7 easyconfigs in this PR)
node2099.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/a27c85275e040fcb385d for a full test report.

**Comment by boegel** [comment]
@wpoely86: any idea why the sanity check failed in your last test?

**Comment by wpoely86** [comment]
@boegel nope, I rebuild it on the same machine and then it worked...

**Pull request merged**